### DISCUSSION TITLE
Distributing idom on conda-forge

### DISCUSSION THREAD

**Discussion by maximlt** [idom-team/idom#850] [Ideas]
Hi all,

I'd be nice to have `idom` available on conda-forge. Adding what is called a *recipe* to conda-forge is [pretty well documented](https://conda-forge.org/#add_recipe). That would ultimately make `pyecharts` directly available to conda users through the community-maintained *conda-forge* channel.

**Comment by rmorshea** [comment]
I created #851 to track this. Out of curiosity, what other Conda packages would you like to use IDOM with? I'm guessing `pyecharts` given your typo, but are there more?

**Reply by maximlt** [reply]
Arf damned copy/paste :) I indeed opened a similar issue/discussion for PyEcharts. My original motivation was that I'm one of the maintainers of [Panel](https://github.com/holoviz/panel), our CI uses conda to install our runtime and optional dependencies, idom (and pyecharts as you can guess) being one of them. It's not really a problem for us, we just pip install it, it falls in the nice-to-have category. As for other conda users in general, they can pip install anything but their workflow (currently) works best when they can conda install everything.

**Comment by shawncrawley** [comment]
I'm in the process of adding reactpy, reactpy-router, and reactpy-django to the conda-forge channel. In fact, I already have [the PR submitted to the conda-forge/staged-recipes repo](https://github.com/conda-forge/staged-recipes/pull/27994). Unfortunately, the automatic build checks are failing due to the same `sdist` issue being addressed by PR #1191, which is still unmerged. Once that gets merged, I'll tackle the remaining roadblocks, if any.

**Reply by Archmonger** [reply]
I'll take a look at that PR soon, looks like it'll need some effort to be mergeable.
